reactjs/src/pages/blocks/leftnav.js

```diff
@@ -17,6 +62,13 @@ export default function LeftNav() {
                         <NavLink to={link.path}>{link.pageName}</NavLink>
                     </li>
                 ))}
+
+                <div style={{ position: 'relative', width: '600px', height: '800px' }}>    
+                    <div style={{ position: 'absolute', bottom: '33%', fontStyle: 'italic' }}>
```

Comment:
Can this be replaced with bootstrap classes?  What's being achieved by using absolute bottom 33%?

Instead of using fontStyle italic, bootstrap has a class for that.  https://getbootstrap.com/docs/5.0/utilities/text/#font-weight-and-italics  `fst-italic`